src/project/qml/MuseScore/Project/internal/ScoresPage/ScoreItem.qml

```diff
@@ -39,10 +39,12 @@ FocusScope {
     property bool isNoResultFound: false
     property bool isCloud: false
     property int cloudScoreId: 0
+    property bool isRecentView: false
```

Comment:
I would prefer to give this property a more generic name, like `isRemovingScoresAllowed`. Reason: it is generally not nice to give a base component knowledge about its derived components.

Reply:
I understand. I will change it.